src/pages/CommitDetailPage/CommitCoverage/CommitCoverage.test.jsx

```diff
@@ -406,12 +413,10 @@ describe('CommitCoverage', () => {
     }
   ) {
     const queryClient = new QueryClient({
-      defaultOptions: {
-        queries: {
-          retry: false,
-          suspense: true,
-        },
-      },
+      defaultOptions: { queries: { retry: false, suspense: true } },
+    })
+    const queryClientV5 = new QueryClientV5({
+      defaultOptions: { queries: { retry: false, suspense: true } },
```

Comment:
Do we want this to be `suspense: false` and then rely on using `useSuspenseQueryV5` for anything that explicitly needs suspense ? Just making sure I understand how I should be using the new patterns

Reply:
No, `suspense` isn't a valid option here, JS not catching it tho 😢

Reply:
Ah okay great, see that you removed it now - great just making sure I understood all the pieces for the new client - thanks again for doing that!